src/plugins/english/novelbin.ts

```diff
@@ -7,8 +7,8 @@ class NovelBin implements Plugin.PluginBase {
   id = 'novelbin';
   name = 'Novel Bin';
   icon = 'src/en/novelbin/icon.png';
-  site = 'https://binnovel.com/';
-  version = '1.0.1';
+  site = 'https://lightnovel.novelupdates.net/';
```

Comment:
This site redirects me to https://novelbin.com/

Reply:
So I think you should use `https://lightnovel.novelupdates.net/` for building chapter url instead of changing the domain